docs/Projects/TAD/ADRs/0004-software-stack.md

```diff
@@ -0,0 +1,81 @@
+# ADR-0004 Software Stack for TAD
+
+## Context
+
+For building our own TAD solution, we need to choose a software stack. During our earlier
+POCs and market research, we gathered insight and information on technologies to use
+and which not to use.
+
+During further discussions and brainstorm sessions, a software stack was chosen
+that accommodates our needs best.
+
+While more fine grained [requirements](../Existing-Tools/Comparison/requirements) are listed elsewhere,
+some key requirements are:
+
+* the tool is runnable on local dev machines
+* the tool is runnable as user as a local Docker solution
+* the tool is runnable on a cloud platform based on Kubernetes
+* the tool supports multiple organizations, teams and projects
+* the tool supports oath / keycloak
+
+## Assumptions
+
+We stick to one programming language to avoid unneeded complexity. As most AI related
+tooling is written in Python, this is the logical choice for our development as well.
+
+There is no need for a complex or separate web GUI framework. Also to avoid more components and complexity,
+it is preferred to bundle backend and frontend in one solution.
+
+As a government organization, we need to adhere to the digital inclusion standards, like
+[Drempelvrij](https://www.drempelvrij.nl/).
```

Comment:
Shouldn't we refer (also, or in the first place) to this: https://www.digitoegankelijk.nl/ ?

Reply:
And https://wcag.nl/kennis/richtlijnen/wcag-2-1-richtlijnen/

Reply:
```suggestion
[Digitale toegankelijkheid](https://www.digitoegankelijk.nl/) and [WCAG Guidelines](https://wcag.nl/kennis/richtlijnen/wcag-2-1-richtlijnen/) .
```